### PR TITLE
Fix samples comments

### PR DESCRIPTION
**Pull request by tsurdilo**
Signed-off-by: Tihomir Surdilovic <tihomir@temporal.io>

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Comment by tsurdilo** [comment]
@flossypurse here are the fixes for outstanding comments you made in https://github.com/temporalio/samples-java/pull/114

File: src/main/java/io/temporal/samples/hello/HelloActivity.java [hunk 1]
```diff
@@ -32,13 +32,7 @@
...
 import java.time.Duration;
 
-/**
- * Sample Temporal workflow that executes a single workflow activity.
- *
- * <p>To execute this example a locally running Temporal service instance is required. You can
- * follow instructions on how to set up your Temporal service here:
- * https://github.com/temporalio/temporal/blob/master/README.md#download-and-start-temporal-server-locally
- */
+/** Sample Temporal workflow that executes a single workflow activity. */
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloActivity.java:35]:
```suggestion
/** Sample Temporal Workflow Definition that executes a single Activity. */
```

File: src/main/java/io/temporal/samples/hello/HelloActivityExclusiveChoice.java [hunk 2]
```diff
@@ -33,13 +33,7 @@
...
 import java.util.Map;
 
-/**
- * Sample Temporal workflow demonstrating how to run an activity based on dynamic input.
- *
- * <p>To execute this example a locally running Temporal service instance is required. You can
- * follow instructions on how to set up your Temporal service here:
- * https://github.com/temporalio/temporal/blob/master/README.md#download-and-start-temporal-server-locally
- */
+/** Sample Temporal workflow demonstrating how to run an activity based on dynamic input. */
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloActivityExclusiveChoice.java:36]:
```suggestion
/** Sample Temporal Workflow Definition demonstrating how to execute an Activity based on dynamic input. */
```

File: src/main/java/io/temporal/samples/hello/HelloAsync.java [hunk 3]
```diff
@@ -33,13 +33,7 @@
...
 import java.time.Duration;
 
-/**
- * Sample Temporal workflow that demonstrates asynchronous activity invocation.
- *
- * <p>To execute this example a locally running Temporal service instance is required. You can
- * follow instructions on how to set up your Temporal service here:
- * https://github.com/temporalio/temporal/blob/master/README.md#download-and-start-temporal-server-locally
- */
+/** Sample Temporal workflow that demonstrates asynchronous activity invocation. */
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloAsync.java:36]:
```suggestion
/** Sample Temporal Workflow Definition that demonstrates  an asynchronous Activity Execution. */
```

File: src/main/java/io/temporal/samples/hello/HelloAsyncActivityCompletion.java [hunk 4]
```diff
@@ -34,13 +34,7 @@
...
 import java.util.concurrent.ForkJoinPool;
 
-/**
- * Sample Temporal workflow that demonstrates asynchronous activity implementation.
- *
- * <p>To execute this example a locally running Temporal service instance is required. You can
- * follow instructions on how to set up your Temporal service here:
- * https://github.com/temporalio/temporal/blob/master/README.md#download-and-start-temporal-server-locally
- */
+/** Sample Temporal workflow that demonstrates asynchronous activity implementation. */
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloAsyncActivityCompletion.java:37]:
```suggestion
/** Sample Temporal Workflow Definition that demonstrates asynchronous Activity Execution */
```

File: src/main/java/io/temporal/samples/hello/HelloAsyncActivityCompletion.java [hunk 5]
```diff
@@ -167,44 +162,42 @@ private void composeGreetingAsync(byte[] taskToken, String greeting, String name
...
      */
     Worker worker = factory.newWorker(TASK_QUEUE);
 
-    /*
-     * Register our workflow implementation with the worker.
-     * Workflow implementations must be known to the worker at runtime in
-     * order to dispatch workflow tasks.
+    /**
+     * Register our workflow implementation with the worker. Workflow implementations must be known
+     * to the worker at runtime in order to dispatch workflow tasks.
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloAsyncActivityCompletion.java:187]:
```suggestion
     * Register our Workflow Types with the Worker. Workflow Types must be known
     * to the Worker at runtime in order for it to poll for Workflow Tasks.
```

File: src/main/java/io/temporal/samples/hello/HelloAsyncActivityCompletion.java [hunk 6]
```diff
@@ -167,44 +162,42 @@ private void composeGreetingAsync(byte[] taskToken, String greeting, String name
...
      */
     ActivityCompletionClient completionClient = client.newActivityCompletionClient();
     worker.registerActivitiesImplementations(new GreetingActivitiesImpl(completionClient));
 
-    /*
-     * Start all the workers registered for a specific task queue.
-     * The started workers then start polling for workflows and activities.
+    /**
+     * Start all the workers registered for a specific task queue. The started workers then start
+     * polling for workflows and activities.
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloAsyncActivityCompletion.java:200]:
```suggestion
     * Start all the Workers registered for a specific Task Queue. The Workers then start
     * polling for Workflow Tasks and Activity Tasks.
```

File: src/main/java/io/temporal/samples/hello/HelloAsyncLambda.java [hunk 7]
```diff
@@ -36,10 +36,6 @@
 /**
  * Sample Temporal workflow that demonstrates asynchronous invocation of multiple workflow
  * activities.
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloAsyncLambda.java:38]:
```suggestion
 * Sample Temporal Workflow Definition that demonstrates an asynchronous Activity Executions.
```

File: src/main/java/io/temporal/samples/hello/HelloCancellationScope.java [hunk 8]
```diff
@@ -48,10 +48,6 @@
  * Sample Temporal workflow that demonstrates parallel invocation of multiple workflow activities
  * with cancellation scope. When one of the invoked activities finishes, we cancel execution of the
  * other activities and wait for their cancellation to complete.
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloCancellationScope.java:50]:
```suggestion
 * Sample Temporal Workflow Definition that demonstrates parallel Activity Executions
 * with a Cancellation Scope. When one of the Activity Executions finish, we cancel the execution of the
 * other Activities and wait for their cancellation to complete.
```

File: src/main/java/io/temporal/samples/hello/HelloChild.java [hunk 9]
```diff
@@ -34,10 +34,6 @@
  * Sample Temporal workflow that demonstrates use of a child workflow. Child workflows allow you to
  * group your workflow logic into small logical and reusable units that solve a particular problem.
  * They can be typically reused by multiple other workflows.
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloChild.java:36]:
```suggestion
 * Sample Temporal Workflow Definition that demonstrates the execution of a Child Workflow. Child workflows allow you to
 * group your Workflow logic into small logical and reusable units that solve a particular problem.
 * They can be typically reused by multiple other Workflows.
```

File: src/main/java/io/temporal/samples/hello/HelloPolymorphicActivity.java [hunk 10]
```diff
@@ -34,10 +34,6 @@
 /**
  * Sample Temporal workflow that demonstrates the use of multiple activities which extend a common
  * interface.
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloPolymorphicActivity.java:36]:
```suggestion
 * Sample Temporal Workflow Definition that demonstrates the execution of multiple Activities which extend a common
 * interface.
```

File: src/main/java/io/temporal/samples/hello/HelloQuery.java [hunk 11]
```diff
@@ -30,13 +30,7 @@
...
 import java.time.Duration;
 
-/**
- * Sample Temporal workflow that demonstrates the workflow query capability.
- *
- * <p>To execute this example a locally running Temporal service instance is required. You can
- * follow instructions on how to set up your Temporal service here:
- * https://github.com/temporalio/temporal/blob/master/README.md#download-and-start-temporal-server-locally
- */
+/** Sample Temporal workflow that demonstrates the workflow query capability. */
```
Comment (flossypurse) [inline src/main/java/io/temporal/samples/hello/HelloQuery.java:33]:
```suggestion
/** Sample Temporal Workflow Definition that demonstrates how to Query a Workflow. */
```

**Review by flossypurse — APPROVED** [review]
Overall pretty good improvement.

**Comment by tsurdilo** [comment]
@flossypurse thanks for review! applied all suggestions

**Pull request merged**